corehq/apps/data_analytics/models.py

```diff
@@ -157,3 +157,94 @@ def export_row(self, past_months):
             experienced_threshold=self.experienced_threshold or DEFAULT_EXPERIENCED_THRESHOLD,
             performance_threshold=self.performance_threshold or DEFAULT_PERFORMANCE_THRESHOLD,
         )
+
+
+class DomainMetrics(models.Model):
+
+    domain = models.TextField(unique=True, db_index=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    last_modified = models.DateTimeField(auto_now=True)
+
+    first_domain_for_creating_user = models.BooleanField()
+    has_project_icon = models.BooleanField()
+    has_security_settings = models.BooleanField()
+    is_active = models.BooleanField()
+
+    lookup_tables = models.IntegerField()
+    repeaters = models.IntegerField()
+    ucrs = models.IntegerField()
+
+    # App Metrics
+    apps = models.IntegerField()
+    apps_with_icon = models.IntegerField()
+    apps_with_multiple_languages = models.IntegerField()
+
+    # User Metrics
+    mobile_workers = models.IntegerField()
+    web_users = models.IntegerField()
+
+    active_mobile_workers = models.IntegerField()
+    active_mobile_workers_in_past_year = models.IntegerField()
+    case_sharing_groups = models.IntegerField()
+    case_sharing_locations = models.IntegerField()
+    has_custom_roles = models.BooleanField()
+    has_locations = models.BooleanField()
+    location_restricted_users = models.IntegerField()
+    users_with_submission = models.IntegerField()
+
+    # Case Metrics
+    cases = models.IntegerField()
+
+    active_cases = models.IntegerField()
+    cases_modified_in_last_30_days = models.IntegerField()
+    cases_modified_in_last_60_days = models.IntegerField()
+    cases_modified_in_last_90_days = models.IntegerField()
+    inactive_cases = models.IntegerField()
+    user_cases_modified_in_last_30_days = models.IntegerField()
```

Comment:
It seems like we normally use `usercase` without the underscore (even if the previous calced prop used `user_cases`): https://github.com/dimagi/commcare-hq/blob/6e52bf2cca52537238fc7461503e390ccf14cb93/corehq/apps/app_manager/const.py#L22-L24

Reply:
467a3663ab8330c27762a5e26ebd33c7ce1d687d